Tools/CISupport/ews-build/events.py

```diff
@@ -315,26 +315,36 @@ def _get_commit_msg(self, repo, sha):
 
     @defer.inlineCallbacks
     def _get_pr_files(self, repo, number):
-        # Copied from https://github.com/buildbot/buildbot/blob/v2.10.5/master/buildbot/www/hooks/github.py to include added/modified/deleted
-        headers = {"User-Agent": "Buildbot"}
```

Comment:
we don't need User-Agent anymore?

Reply:
Because our `TwistedAdditions` code handles that for us.